components/camel-elasticsearch5/src/main/java/org/apache/camel/component/elasticsearch5/ElasticsearchProducer.java

```diff
@@ -208,37 +210,53 @@ public void process(Exchange exchange) throws Exception {
         }
 
     }
-    
+
     @Override
     @SuppressWarnings("unchecked")
     protected void doStart() throws Exception {
         super.doStart();
 
         if (client == null) {
             LOG.info("Connecting to the ElasticSearch cluster: " + configuration.getClusterName());
-            
             if (configuration.getIp() != null) {
-                client = new PreBuiltTransportClient(getSettings())
+                client = createClient()
                     .addTransportAddress(new InetSocketTransportAddress(InetAddress.getByName(configuration.getIp()), configuration.getPort()));
             } else if (configuration.getTransportAddressesList() != null
-                    && !configuration.getTransportAddressesList().isEmpty()) {
+                && !configuration.getTransportAddressesList().isEmpty()) {
                 List<TransportAddress> addresses = new ArrayList<TransportAddress>(configuration.getTransportAddressesList().size());
                 for (TransportAddress address : configuration.getTransportAddressesList()) {
                     addresses.add(address);
                 }
-                client = new PreBuiltTransportClient(getSettings()).addTransportAddresses(addresses.toArray(new TransportAddress[addresses.size()]));
+                client = createClient().addTransportAddresses(addresses.toArray(new TransportAddress[addresses.size()]));
             } else {
                 LOG.info("Incorrect ip address and port parameters settings for ElasticSearch cluster");
             }
         }
     }
 
-    private Settings getSettings() {
-        return Settings.builder()
-                .put("cluster.name", configuration.getClusterName())
-                .put("client.transport.ignore_cluster_name", false)
-                .put("client.transport.sniff", configuration.getClientTransportSniff())
-                .build();
+    private TransportClient createClient() {
+        final Settings.Builder settings = getSettings();
+        try {
+            Class clazz = Class.forName("org.elasticsearch.xpack.client.PreBuiltXPackTransportClient");
+            Constructor<?> ctor = clazz.getConstructor(Settings.class, Class[].class);
+            settings.put("xpack.security.user", configuration.getUser() + ":" + configuration.getPassword())
+                .put("xpack.security.transport.ssl.enabled", configuration.getEnableSSL());
+            return (TransportClient) ctor.newInstance(new Object[] {settings.build(), new Class[0]});
+        } catch (Exception e) {
+            LOG.info("XPack Client was not found on the classpath, using the standard client");
```

Comment:
Should maybe be in DEBUG mode as most people will not use XPack Client. Instead you revert this and do an INFO logging if you can create the XPack Client